modules/API/external/external_api.js

```diff
@@ -207,11 +206,32 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
      * @private
      */
     _createIFrame(height, width) {
+        // Compute valid values for height and width. If a number is specified
+        // it's treated as pixel units and our minimum constraints are applied.
+        // If the value is expressed in em, pt or percentage, it's used as is.
+        // Also protect ourselves from undefined, because
+        // Math.max(undefined, 100) === NaN, obviously.
+        const re = /([0-9]*\.?[0-9]+)(em|pt|%)$/;
```

Comment:
mind giving an example of what the regex matches?  i find that makes reading regex's a lot easier since they're practically a write-only syntax :)